### PR TITLE
css introduced to monitorix.cgi file

### PR DESCRIPTION
**Pull request by z-eos**
monitorix.cgi, system.pm and .css files

### CODE REVIEW

File: monitorix.cgi [hunk 1]
```diff
@@ -448,6 +436,11 @@ if($RRDs::VERSION > 1.2) {
 	if($RRDs::VERSION >= 1.3) {
 		push(@version12_small, "--font=DEFAULT:0:Mono");
 	}
+
+	push(@version12, "--alt-autoscale", "--allow-shrink" );
+	push(@version12_small, "--alt-autoscale", "--allow-shrink" );
+
+
 }
```
Comment (mikaku) [inline monitorix.cgi:444]: Why do you need to include these lines?

**Comment by mikaku** [comment]
Also,

The outset and inset colors must be touched again.

I know that this change will affect the main page outset and inset colors but it doesn't matter, this 3D effect is much better than the main page.

In the following screen shot:
- Top belongs to the original (perfect 3D effect).
- Bottom belongs to the CSS.

Untitled

File: monitorix.cgi (hunk 1 above)
Comment (z-eos) [reply monitorix.cgi:444]: just a proposal, charts looks better

Comment (mikaku) [reply monitorix.cgi:444]: No, they are shifted to the top and there is a useless gap in the bottom.
Monitorix already has a specific option to include extra RRDtool arguments.

This change is not acceptable.

**Comment by z-eos** [comment]
> I know that this change will affect the main page outset and inset colors but it doesn't matter, this 3D effect is much better than the main page.

no, they use different classes and can be managed separately

**Comment by mikaku** [comment]
You can use the same class as the 3D effect here is better than in the main page.

**Comment by mikaku** [comment]
The 3D effect is incomplete.
Please, compare it with the screen shot above.

**Comment by mikaku** [comment]
The `system.pm` graph does even have such 3D effect.

File: monitorix.cgi [hunk 2]
```diff
@@ -448,6 +436,8 @@ if($RRDs::VERSION > 1.2) {
 	if($RRDs::VERSION >= 1.3) {
 		push(@version12_small, "--font=DEFAULT:0:Mono");
 	}
+
+
```
Comment (mikaku) [inline monitorix.cgi:440]: Please, remove these blank lines that are left after you removed the extra RRDtool arguments.

Comment (z-eos) [reply monitorix.cgi]: done

**Pull request merged**